demos/asp-net-core/ITOps.ViewModelComposition/CompositionHandler.cs

```diff
@@ -0,0 +1,57 @@
+﻿using ITOps.ViewModelComposition.Engine;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.DependencyInjection;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ITOps.ViewModelComposition.Gateway
+{
+    public class CompositionHandler
+    {
+        public static async Task<(dynamic ViewModel, int StatusCode)> HandleGetRequest(HttpContext context)
```

Comment:
It's used by the `CompositionActionFilter` as well, in the MVC project. This is the main reason I was reluctant to split the solution and make 2 with shared projects, it's impossible to see the whole picture.